05-regression.Rmd

```diff
@@ -122,7 +122,7 @@ library(patchwork)
 
 Why do some professors and instructors at universities and colleges receive high teaching evaluations scores from students while others receive lower ones? Are there differences in teaching evaluations between instructors of different demographic groups? Could there be an impact due to student biases? These are all questions that are of interest to university/college administrators, as teaching evaluations are among the many criteria considered in determining which instructors and professors get promoted.
 
-Researchers at the University of Texas in Austin, Texas (UT Austin) tried to answer the following research question: what factors explain differences in instructor teaching evaluation scores? To this end, they collected instructor and course information on 463 courses. A full description of the study can be found at [openintro.org](https://www.openintro.org/stat/data/?data=evals).
+Researchers at the University of Texas in Austin, Texas (UT Austin) tried to answer the following research question: what factors explain differences in instructor teaching evaluation scores? To this end, they collected instructor and course information on `r nrow(moderndive::evals)` courses. A full description of the study can be found at [openintro.org](https://www.openintro.org/stat/data/?data=evals).
```

Comment:
Same comment regarding dimensions of `drinks_smaller_tidy`. Let's compute this value only once right after `evals` is loaded and save in an interval variable called `n_courses`